db2i_charsetSupport.cc

```diff
@@ -574,161 +244,6 @@ int32 getEncodingScheme(const uint16 inCcsid, int32& outEncodingScheme)
   DBUG_RETURN(0);
 }
 
-
-/**
-  Get the best fit equivalent CCSID. (Wrapper for QTQGRDC API)
-    
-  @param inCcsid  An IBM i CCSID
-  @param inEncodingScheme  The encoding scheme
-  @param[out] outCcsid  The equivalent CCSID
-    
-  @return  0 if successful. Failure otherwise
-*/
-int32 getAssociatedCCSID(const uint16 inCcsid, const int inEncodingScheme, uint16* outCcsid)
```

Comment:
This function was only used in `convertIANAToDb2Ccsid` and iit has been removed.